README.md

```diff
@@ -37,7 +37,20 @@ end
 
 After checking out the repo, run `bin/setup` to install dependencies. Then, run `rake spec` to run the tests. You can also run `bin/console` for an interactive prompt that will allow you to experiment.
 
-To install this gem onto your local machine, run `bundle exec rake install`. To release a new version, update the version number in `version.rb`, and then run `bundle exec rake release`, which will create a git tag for the version, push git commits and tags, and push the `.gem` file to [rubygems.org](https://rubygems.org).
+To install this gem onto your local machine, run `bundle exec rake install`.
+
+## Releasing
```

Comment:
These instructions assume that the publisher is a member of the Retirable team. Should I develop separate instructions for external contributors?

Reply:
no I think this is great for now — I don't think we'd let external contributors publish changes, so they'd just merge the approved PR and the version would get updated by a member of the Retirable engineering team.